antipodal_gripper_controller/doc/userdoc.rst

```diff
@@ -0,0 +1,14 @@
+:github_url: https://github.com/ros-controls/ros2_controllers/blob/{REPOS_FILE_BRANCH}/antipodal_gripper_controller/doc/userdoc.rst
+
+.. _antipodal_gripper_controller_userdoc:
+
+Antipodal Gripper Action Controller
+-----------------------------------
+
+Controller for executing a ``AntipodalGripperCommand`` action for simple antipodal grippers.
```

Comment:
Yes some more documentation would be helpful. It would also be nice to add some gripper manufacturers that are of antipodal type. It would be easier for the people to connect